run_checks_and_format.py

```diff
@@ -30,7 +31,12 @@
 
 def run_process(cmd_list):
     """Run popen process."""
-    p = Popen(cmd_list, stdout=PIPE, stderr=PIPE)
+
+    try:
+        p = Popen(cmd_list, stdout=PIPE, stderr=PIPE)
+    except WindowsError:
```

Comment:
This is an specific windows error, so unless this only runs on windows this should be `OSError`